infra/docker-swarm/README.md

```diff
@@ -0,0 +1,105 @@
+# Setting up a Docker Swarm cluster on AWS for Testground
+
+In this directory, you will find:
+
+```
+» tree -L 1
+.
+├── README.md
+├── ansible     # Playbooks used to configure each of the instances
+└── terraform   # Playbooks used to setup AWS infrastructure - EC2 instances, security groups, networks, etc.
+```
+
+## Requirements
+
+- 1. [Terraform](https://www.terraform.io/).
+- 2. [Ansible](https://www.ansible.com/).
+- 3. [op](https://support.1password.com/command-line-getting-started/).
+- 4. [terraform-inventory](https://github.com/adammck/terraform-inventory).
+
+## Set up infrastructure with Terraform
+
+The playbooks set up a "manager" machine, another machine to run Redis, and two worker machines for running tests on.
+
+1. Configure your AWS credentials
```

Comment:
Using the AWS cli. Add a link to https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html

Reply:
Added link.